pages/badges/badge-single.jsx

```diff
@@ -16,16 +16,19 @@ var React = require('react'),
     Modal = require('../../components/modal.jsx');
 
 var Navigation = React.createClass({
+  contextTypes: {
+    intl: React.PropTypes.object
+  },
   render: function() {
     var prev = this.props.prev;
     var next = this.props.next;
     return (
       <div className="badge-navigation">
-        <a className="previous" href={prev.url}>
+        <a className="previous" href={"/" + this.context.intl.locale + prev.url}>
```

Comment:
Reading this code makes it feels like it would be very sensible to capture this value as a `this.locale`, or better yet, since `locale` doesn't trickle down but is static for an entire site, having a lib that can act as central locale consultation point... that may be out of scope for this PR, but having to get this value out of a `this.context.intl` namespace seems too elaborate